bftengine/include/bftengine/SimpleClient.hpp

```diff
@@ -40,7 +40,7 @@ struct SimpleClientParams {
 };
 
 // Possible values for 'flags' parameter
-enum ClientMsgFlag : uint8_t {
+enum ClientMsgFlag : uint64_t {
```

Comment:
I think the `SimpleClient.hpp` is not used at all. If so, let's not change it at all.

Reply:
As discussed offline, `SimpleClient.hpp` is still being used at various places in the code (for eg `external_client.hpp` and `concord_client_pool.hpp` import it) and utilise this enum. 
Ideally (as discussed), `SimpleClient.hpp` and its related code should be removed from repo (because its mostly  dead code) and such cases (like enum usage) should be handled gracefully. Will add a tech debt item for it upon discussion with Celine.

Also, reason for upgrading from 8b to 64b for this flag, is to maintain consistency with enum `MsgFlag : uint64_t` in `Replica.hpp` which already is on 64b . We need to have flag consistency across all 3 places changed.